net-libs/xdp-tools/files/1.4.1-fix-memory-leak-in-xsk_setup_xdp_prog.patch

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
This changes the installed program / runtime behaviour and hence needs a new revision.

Reply:
:heavy_check_mark: